devcontainers does not work on Windows due to incorrect file name

### Reproduction steps

1. On Windows, set up any valid `.devcontainer/devcontainer.json`
2. Start the Dev container

NOTE: It works as intended after renaming: 
`C:\Users\USER\AppData\Local\Zed\devcontainer\node_modules\@devcontainers\cli\devcontainer.js`
to
`C:\Users\USER\AppData\Local\Zed\devcontainer\node_modules\@devcontainers\cli\devcontainer.cmd.js`

### Current vs. Expected behavior

The dev container should start

### Zed version and system specs

Zed Preview 0.220.1
OS: Windows 11

### Attach Zed log file

<details><summary>log</summary>
2026-01-15T15:23:27-05:00 ERROR [recent_projects::dev_container] Could not run devcontainer CLI from data_dir. Will try once more to install. Output: Output { status: ExitStatus(ExitStatus(1)), stdout: "", stderr: "node:internal/modules/cjs/loader:1423\r\n  throw err;\r\n  ^\r\n\r\nError: Cannot find module 'C:\\Users\\USER\\AppData\\Local\\Zed\\devcontainer\\node_modules\\@devcontainers\\cli\\devcontainer.cmd.js'\r\n    at Module._resolveFilename (node:internal/modules/cjs/loader:1420:15)\r\n    at defaultResolveImpl (node:internal/modules/cjs/loader:1058:19)\r\n    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1063:22)\r\n    at Module._load (node:internal/modules/cjs/loader:1226:37)\r\n    at TracingChannel.traceSync (node:diagnostics_channel:328:14)\r\n    at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)\r\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)\r\n    at node:internal/main/run_main_module:33:47 {\r\n  code: 'MODULE_NOT_FOUND',\r\n  requireStack: []\r\n}\r\n\r\nNode.js v24.11.0\r\n" }
2026-01-15T15:23:39-05:00 ERROR [recent_projects::dev_container] Non-success status running devcontainer up for workspace: out: "", err: "node:internal/modules/cjs/loader:1423\r\n  throw err;\r\n  ^\r\n\r\nError: Cannot find module 'C:\\Users\\USER\\AppData\\Local\\Zed\\devcontainer\\node_modules\\@devcontainers\\cli\\devcontainer.cmd.js'\r\n    at Module._resolveFilename (node:internal/modules/cjs/loader:1420:15)\r\n    at defaultResolveImpl (node:internal/modules/cjs/loader:1058:19)\r\n    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1063:22)\r\n    at Module._load (node:internal/modules/cjs/loader:1226:37)\r\n    at TracingChannel.traceSync (node:diagnostics_channel:328:14)\r\n    at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)\r\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)\r\n    at node:internal/main/run_main_module:33:47 {\r\n  code: 'MODULE_NOT_FOUND',\r\n  requireStack: []\r\n}\r\n\r\nNode.js v24.11.0\r\n"
2026-01-15T15:23:39-05:00 ERROR [recent_projects] Failed to start Dev Container: DevContainerUpFailed("Failed with nested error: DevContainer creation failed with error: Non-success status running devcontainer up for workspace: out: \"\", err: \"node:internal/modules/cjs/loader:1423\\r\\n  throw err;\\r\\n  ^\\r\\n\\r\\nError: Cannot find module 'C:\\\\Users\\\\USER\\\\AppData\\\\Local\\\\Zed\\\\devcontainer\\\\node_modules\\\\@devcontainers\\\\cli\\\\devcontainer.cmd.js'\\r\\n    at Module._resolveFilename (node:internal/modules/cjs/loader:1420:15)\\r\\n    at defaultResolveImpl (node:internal/modules/cjs/loader:1058:19)\\r\\n    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1063:22)\\r\\n    at Module._load (node:internal/modules/cjs/loader:1226:37)\\r\\n    at TracingChannel.traceSync (node:diagnostics_channel:328:14)\\r\\n    at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)\\r\\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)\\r\\n    at node:internal/main/run_main_module:33:47 {\\r\\n  code: 'MODULE_NOT_FOUND',\\r\\n  requireStack: []\\r\\n}\\r\\n\\r\\nNode.js v24.11.0\\r\\n\"")
</details>


### Relevant Zed settings

<details><summary>settings.json</summary>

<!-- Paste your settings inside the code block. -->
```json

```

</details>


### Relevant Keymap

_No response_

### (for AI issues) Model provider details

_No response_

### If you are using WSL on Windows, what flavor of Linux are you using?

None

--- a/crates/dev_container/src/devcontainer_api.rs
+++ b/crates/dev_container/src/devcontainer_api.rs
@@ -334,6 +334,10 @@ fn dev_container_cli() -> String {
     "devcontainer.cmd".to_string()
 }
 
+fn dev_container_script() -> String {
+    "devcontainer.js".to_string()
+}
+
 async fn check_for_docker(use_podman: bool) -> Result<(), DevContainerError> {
     let mut command = if use_podman {
         util::command::new_smol_command("podman")
@@ -371,7 +375,7 @@ async fn ensure_devcontainer_cli(
             .join("node_modules")
             .join("@devcontainers")
             .join("cli")
-            .join(format!("{}.js", &dev_container_cli()));
+            .join(&dev_container_script());
 
         log::debug!(
             "devcontainer not found in path, using local location: ${}",

